addons/point_of_sale/static/src/app/screens/product_screen/control_buttons/control_buttons.js

```diff
@@ -118,7 +118,7 @@ export class ControlButtons extends Component {
     }
     internalNoteLabel(order) {
         if (order) {
-            return _t("General Note");
+            return _t("Footer Note");
```

Comment:
need to update the translation?

Reply:
Yes because from here only string `General Note` is been passed.